modin/experimental/engines/omnisci_on_ray/test/test_dataframe.py

```diff
@@ -1030,32 +1087,38 @@ def value_counts(df, cols, dropna, sort, ascending, **kwargs):
 
 class TestMerge:
     data = {
-        "a": [1, 2, 3],
-        "b": [10, 20, 30],
-        "e": [11, 22, 33],
+        "a": [1, 2, 3, 6, 5, 4],
+        "b": [10, 20, 30, 60, 50, 40],
+        "e": [11, 22, 33, 66, 55, 44],
     }
     data2 = {
-        "a": [4, 2, 3],
-        "b": [40, 20, 30],
-        "d": [4000, 2000, 3000],
+        "a": [4, 2, 3, 7, 1, 5],
+        "b": [40, 20, 30, 70, 10, 50],
+        "d": [4000, 2000, 3000, 7000, 1000, 5000],
     }
     on_values = ["a", ["a"], ["a", "b"], ["b", "a"]]
```

Comment:
```suggestion
    on_values = ["a", ["a"], ["a", "b"], ["b", "a"], None]
```
and remove `test_default_merge` function

Reply:
done